src/main/java/com/eppo/sdk/helpers/FetchConfigurationsTask.java

```diff
@@ -16,7 +16,8 @@ public class FetchConfigurationsTask extends TimerTask {
   @Override
   public void run() {
     configurationStore.fetchAndSetExperimentConfiguration();
-    long delay = this.intervalInMillis - ((long) Math.random() * this.jitterInMillis);
+    // TODO: retry on failed fetches
+    long delay = this.intervalInMillis - (long)(Math.random() * this.jitterInMillis);
```

Comment:
:broom: We needed to shift around the parentheses as the way they were the jitter was always `0`